### PR TITLE
created WeatherStruct; created request for json

### PR DESCRIPTION
**Pull request by ZhuravlevMax**
Создал запрос погоды.
Создал структуру для конвертации данных из запрошенного файла json

### CODE REVIEW

File: WeatherZhuravlevMax/WeatherZhuravlevMax/ViewController.swift [hunk 1]
```diff
@@ -11,7 +11,36 @@ class ViewController: UIViewController {
 
     override func viewDidLoad() {
         super.viewDidLoad()
-        // Do any additional setup after loading the view.
+
+        //url по которому будем получать данные
+        if let url = URL(string:"https://api.openweathermap.org/data/2.5/weather?q=Minsk&appid=e3d520a7f75cff3164067d89df3e34a8") {
```
Comment (vkhadyka) [inline WeatherZhuravlevMax/WeatherZhuravlevMax/ViewController.swift:16]: It's a bad idea hardcode url params

Comment (ZhuravlevMax) [reply WeatherZhuravlevMax/WeatherZhuravlevMax/ViewController.swift:16]: Do you mean I have to add variable for city?

Comment (vkhadyka) [reply WeatherZhuravlevMax/WeatherZhuravlevMax/ViewController.swift:16]: Yup, but you have to move apiKey to a different project location also. I hope you can recall what I meant.

Comment (ZhuravlevMax) [reply WeatherZhuravlevMax/WeatherZhuravlevMax/ViewController.swift]: I have corrected your note. Can you see it?

**Comment by ZhuravlevMax** [comment]
Fixed mistake with URL;
Add APIKey to infoPlist

File: WeatherZhuravlevMax/WeatherZhuravlevMax/AppDelegate.swift [hunk 2]
```diff
@@ -14,6 +14,8 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
 
     func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
         // Override point for customization after application launch.
+        //Берем ключ из info.plist
+        let key = Bundle.main.object(forInfoDictionaryKey: "APIKey") as? String
```
Comment (vkhadyka) [inline WeatherZhuravlevMax/WeatherZhuravlevMax/AppDelegate.swift:18]: What a reason get the key from `info.plist` in `AppDelegate`?

File: WeatherZhuravlevMax/WeatherZhuravlevMax/ViewController.swift [hunk 3]
```diff
@@ -15,8 +15,12 @@ class ViewController: UIViewController {
         let city = "Minsk"
         let key = Bundle.main.object(forInfoDictionaryKey: "APIKey") as? String
         
+        guard let checkedKey = key else {
```
Comment (vkhadyka) [inline WeatherZhuravlevMax/WeatherZhuravlevMax/ViewController.swift:18]: I believe that you can combine 16 and 18 lines

File: WeatherZhuravlevMax/WeatherZhuravlevMax/AppDelegate.swift (hunk 2 above)
Comment (ZhuravlevMax) [reply WeatherZhuravlevMax/WeatherZhuravlevMax/AppDelegate.swift]: Sorry, just forgot to delete those strings

**Review by vkhadyka — APPROVED** [review]
Good job!
Looks awesome.

**Pull request merged**